core/models/user.py

```diff
@@ -89,6 +89,7 @@ def _create_assignee_object_for_user(self):
 
 class SecurityUser(User):
     badge_number = models.CharField(max_length=30, unique=True)
+    active = models.BooleanField(default=True)
```

Comment:
the User model comes with an `is_active` field. we can use that for consistency.